gammapy/image/maps.py

```diff
@@ -252,6 +253,14 @@ def solid_angle(self):
         omega = -np.diff(xsky, axis=1)[1:, :] * np.diff(ysky, axis=0)[:, 1:]
         return Quantity(omega, 'deg2').to('sr')
 
+    def center(self):
+        """
+        Center coordinates of the sky map.
```

Comment:
I'm not sure either, but as long as as it's only the difference between calling `.center` or `.center()`, I don't care much. I think it only makes a real difference if you'd like to define a special setter for a property as part of the API. E.g. in this case one could imaging, reprojecting (shifting) the map to a new center position, when `.center` is set to a different `SkyCoord`, instead of implementing a method `.recenter()`.